pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCommuterMobility.py

```diff
@@ -344,22 +350,22 @@ def test_commuter_data(self):
         countypop_list = list(population[dd.EngEng['population']])
         self.assertEqual(countypop_list[city_from], 163905)
         self.assertAlmostEqual(
-            mat_commuter_migration.iat[city_from, city_to], 1.526, 2)
+            mat_commuter_migration.iat[city_from, city_to], 0.305, 2)
 
         # check migration from Duisburg to Oberspreewald-Lausitz
         city_from = countykey2numlist['05112']
         city_to = countykey2numlist['12066']
-        self.assertEqual(mat_commuter_migration.iat[city_from, city_to], 11)
+        self.assertEqual(mat_commuter_migration.iat[city_from, city_to], 2.2)
 
         # check migration from Lahn-Dill-Kreis to Hamburg
         city_from = countykey2numlist['06532']
         city_to = countykey2numlist['02000']
-        self.assertEqual(mat_commuter_migration.iat[city_from, city_to], 96)
+        self.assertAlmostEqual(mat_commuter_migration.iat[city_from, city_to], 19.2, 2)
 
         # check migration from Herzogtum Lauenburg to Flensburg, Stadt
         city_from = countykey2numlist['01001']
         city_to = countykey2numlist['01053']
-        self.assertEqual(mat_commuter_migration.iat[city_from, city_to], 14)
+        self.assertEqual(mat_commuter_migration.iat[city_from, city_to], 7)
```

Comment:
Please introduce some value tests where no scaling happened.

Reply:
added in latest commit